pages/Filters-Overview.md

```diff
@@ -154,23 +153,62 @@ filter: function() { return feature.height <= 100; }
 filter: function() { return false; }
 ```
 
-### Feature filters can be combined using boolean functions
+#### Keyword properties
+
+The keyword `$zoom` matches the current zoom level. It can be used with the `min` and `max` functions.
+
+```yaml
+filter: { $zoom: 14 }
 
-The following functions combine one or more feature filters into a new feature filter:
+filter: { $zoom: { min: 10 } }
+
+filter:
+    $zoom: { min: 12, max: 15 }
+```
+
+The keyword `$geometry` matches the feature's geometry type, for cases when a FeatureCollection includes more than one type of kind of geometry.
+
+```yaml
+filter: { $geometry: point }
+
+filter: { $geometry: polygon }
+```
+
+#### Filter functions
+
+The filter functions `min` and `max` are equivalent to `>=` and `<=` in a JavaScript function, and can be used in combination.
```

Comment:
I find it kind of confusing to put 'min' and 'max' together with the Boolean functions because they aren't actually very similar. 'min' and 'max' produce the right-hand side of an equality statement whereas the Boolean functions produce entire statements. It's weird to call both of these things "functions".